### ISSUE TITLE
Improve help message

### ISSUE THREAD

**Issue by lgeiger** [larq/zookeeper#101]
It would be great if the help message could show a summary parameters that can be configured via the CLI.

**Comment by AdamHillier** [comment]
Agreed. It's impossible to show all possible configurable params because e.g. if you change the model then the params `model.*` that can be configured changes. But we can at the very least show top-level params which can be set.
